### DISCUSSION TITLE
License and other information

### DISCUSSION THREAD

**Discussion by wolfmurdere** [DependencyTrack/dependency-track#3495] [Q&A]
Hi there!

I have a project manually created in the tool with all the component
information added, however for some of the components I don't find the
corresponding license from the drop down menu under the Legal tab.

These are the license which are not listed - NXP Semiconductor Software
License Agreement, X ORG Permissive License, SQLite and what option has
to be selected for proprietary components owned by the organization?

Also, if I go on adding the licenses mentioned above directly in to the
SBOM after downloading. when I re-upload the SBOM in the Dependency
Track tool, the file doesn't open and the components are not shown in
the screen.

How can the Supplier/Publisher name can be added to the SBOM, is there
an option available in the tool?

Any pointers on the above....

Thanks!

**Comment by black-snow** [comment]
If it's not in https://spdx.org/licenses/ you probably won't find it in DT.
